wordpress-mobile/WordPress-Android
# [JP Install Full Plugin] Open onboarding when individual plugin site is selected the first time

Fixes #17833 

To test:
1 - Install app and make sure the feature flag `JetpackInstallFullPluginFeatureConfig` is enabled;
2 - Select a site with only individual plugins: the Jetpack install full plugin onboarding should be shown;
3 - Select a different site and move back to the site with only individual plugins: the onboarding dialog should not be shown.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and unit testing.

3. What automated tests I added (or what prevented me from doing so)
`JetpackFullPluginInstallOnboardingViewModelTest.kt` and `GetShowJetpackFullPluginInstallOnboardingUseCaseTest.kt`.

PR submission checklist:
